src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs

```diff
@@ -33,6 +33,14 @@ public bool IsWin8OrLater
             }
         }
 
+        public bool Is64BitHost
+        {
+            get
+            {
+                return string.Equals("AMD64", Environment.GetEnvironmentVariable("PROCeSSOR_ARCHITECTURE"), StringComparison.OrdinalIgnoreCase);
```

Comment:
Can we use `RuntimeInformation` here instead: https://docs.microsoft.com/en-us/dotnet/core/api/system.runtime.interopservices.runtimeinformation#System_Runtime_InteropServices_RuntimeInformation_ProcessArchitecture